documentation/3_Layers_of_the_Reference_Architecture_Model/3_2_Functional_Layer/3_2_FunctionalLayer.md

```diff
@@ -137,16 +137,19 @@ given in the [Process Layer](../3_4_Process_Layer/3_4_Process_Layer.md).
 
 #### Vocabularies ####
 
-To create and structure metadata, the operator of a Connector may use
-vocabularies. In doing so, an operator of a Connector can use existing
-vocabularies, create own vocabularies, or work with other operators on
-new vocabularies provided by vocabulary hubs. Vocabulary hubs are
-central servers that store vocabularies and enable collaboration.
-Collaboration may comprise search, selection, matching, updating,
-requests for changes, version management, deletion, duplicate
-identification, and unused vocabularies. Vocabulary hubs need to be
-managed. More information about vocabularies is given in the
-[Information Layer](../3_3_Information_Layer/3_3_InformationLayer.md).
+To create and structure metadata, the data provider in a data space may use vocabularies to be able to tell potential consumers of his data assets the semantics of the various data elements in the data assets at a universal location in a standard electronic format.
+With help of vocabularies the data consumer has possibility to easily understand of the semantics of the various data elements in an offered data asset before downloading it and verify the data asset afterwards.
```

Comment:
With vocabularies the data consumer has the possibility to easily understand the semantics of the various data elements in an offered data asset and is able to verify the data asset afterwards.